### ISSUE TITLE
health check fail

### ISSUE THREAD

**Issue by dullaxe** [derekhe/msfs2020-google-map#121]
I have tried most 3 recent versions, I keep geting health check failed,  running not properly. also tried close Norton anti virus, didn't help

any suggestions ?  thanks

**Comment by derekhe** [comment]
Please give me screen shots when you run the app

**Comment by Predeactor** [comment]
Hi there, I got the same error today when trying to run the program.
I tried turning off my firewall, however this does not change my error, I tried changing the map server, same error too.

Here is my traceback:

```py
Traceback (most recent call last):
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\util\connection.py", line 96, in create_connection
    raise err
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\util\connection.py", line 86, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connectionpool.py", line 382, in _make_request
    self._validate_conn(conn)
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connectionpool.py", line 1010, in _validate_conn
    conn.connect()
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connection.py", line 358, in connect
    conn = self._new_conn()
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x000002046CE33460>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée

Traceback (most recent call last):
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\util\connection.py", line 96, in create_connection
    raise err
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\util\connection.py", line 86, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connectionpool.py", line 382, in _make_request
    self._validate_conn(conn)
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connectionpool.py", line 1010, in _validate_conn
    conn.connect()
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connection.py", line 358, in connect
    conn = self._new_conn()
  File "C:\Users\proju\DOWNLO~1\Flight Simulator\GMapsReplacer\Python39\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x000002046CE33460>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée
```
TL;DR The computer refused the request by himself.

**Comment by dullaxe** [comment]
> Please give me screen shots when you run the app
![Screenshot 2021-11-27 112032](https://user-images.githubusercontent.com/95103998/143719677-d76b9a65-d9df-4e0c-b9e3-e29e5df76fed.jpg)

**Comment by dullaxe** [comment]
thank you

**Comment by Predeactor** [comment]
(I confirm the traceback is the same as me, as I thought :))

**Comment by derekhe** [comment]
Please try latest version: https://github.com/derekhe/msfs2020-google-map/releases/tag/2.3.2

**Comment by Predeactor** [comment]
> Checking mock server health
127.0.0.1 - - [28/Nov/2021 12:36:58] "GET /health HTTP/1.0" 200 -
Mock server health passed
Checking nginx server health
Health check passed

👍 Good for me

**Comment by dullaxe** [comment]
> Please try latest version: https://github.com/derekhe/msfs2020-google-map/releases/tag/2.3.2

thanks for the effort, Derek,  but it still doesn't work
![Screenshot 2021-11-28 083109](https://user-images.githubusercontent.com/95103998/143777120-40581e07-43c3-478d-bec4-4ee861fe3243.jpg)
rk
